pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -1568,3 +1594,148 @@ func TestUserPrivileges(t *testing.T) {
 		jobutils.WaitForJobToPause(t, dbA, jobAID)
 	})
 }
+
```

Comment:
nit: this test could use a more descriptive name. Perhaps TestEndToEndDLQ and a short sentence description?